slices/openvswitch-common.yaml

```diff
@@ -0,0 +1,23 @@
+package: openvswitch-common
+
+slices:
+  core:
+    essential:
+      - coreutils_bins
+      - sed_bins
+      - dash_bins
```

Comment:
Are these needed by the files in this slice?

Reply:
yep - ovs-lib is a script and very much depends on a) a functioning shell and b) misc other utilities for parsing arguments and figuring out what todo.